src/NuGet.Core/NuGet.Build.Tasks/NuGet.RestoreEx.targets

```diff
@@ -0,0 +1,24 @@
+<!--
+***********************************************************************************************
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <UsingTask TaskName="NuGet.Build.Tasks.RestoreTaskEx" AssemblyFile="$(RestoreTaskAssemblyFile)" />
+
+  <Target Name="Restore">
+    <RestoreTaskEx
+        DisableParallel="$(RestoreDisableParallel)"
+        Force="$(RestoreForce)"
+        ForceEvaluate="$(RestoreForceEvaluate)"
+        HideWarningsAndErrors="$(HideWarningsAndErrors)"
+        IgnoreFailedSources="$(RestoreIgnoreFailedSources)"
+        Interactive="$([MSBuild]::ValueOrDefault('$(NuGetInteractive)', '$(MSBuildInteractive)'))"
```

Comment:
neat :D 
Has this always been there/

Reply:
I think its been around since MSBuild 12
https://docs.microsoft.com/en-us/visualstudio/msbuild/property-functions?view=vs-2019#msbuild-valueordefault